### ISSUE TITLE
Get rid of winint.dll dependency and use to the new Socket functions instead

### ISSUE THREAD

**Issue by sirtoobii** [sirtoobii/webrequest2#1]
This is basically just a reminder for myself and anybody who intends to use this software.
https://www.mql5.com/en/docs/network

Feel free to contribute if you want :+1:

**Comment by sirtoobii** [comment]
Took a deeper look at this: Unfortunately, the new Socket functions do not allow us to control certificate verification and therefore we have to stick to the _ugly_ dll import.

**Comment by sirtoobii** [comment]
Yes, self signed certificates - but I agree, thanks to Let's Encrypt, this is not a bit issue anymore. However, as there are currently no Metatrader related projects on my side, I don´t see me implementing this in the near future. So If you´re up for the task, I´m happy to review :)

**Comment by Th0masL** [comment]
Yeah, let's keep this issue open and I'll try to have a look when I have time :)
